scripts/addFeed.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+#### according to `start.sh`, ALICE is the Module Owner and will add BOB as a Feed Owner. #####
+./scripts/start.sh > "$(pwd)"/chainlinkd.log 2>&1 &
+sleep 10
+
+aliceAddr=$(chainlinkd keys show alice -a)
+alicePK=$(chainlinkd keys show alice -p)
+
+bobAddr=$(chainlinkd keys show bob -a)
+bobPK=$(chainlinkd keys show bob -p)
+
+cerloAddr=$(chainlinkd keys show cerlo -a)
+cerloPK=$(chainlinkd keys show cerlo -p)
+
+# aDd NeW fEeD bY aLiCe
```

Comment:
what's going on here with the case?

Reply:
woops, i was just in a frustrated mood LOL